cli/lambda-node-version-update.md

```diff
@@ -1,4 +1,57 @@
 # Node Version Update
+
+## Node.js 8.10 to Node.js 10.x
+According to AWS Lambda [Runtime Support Policy](https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html), AWS Lambda deprecates Node.js 
+runtime Node.js 8.10 on January 6th, 2020. 
+
+The Amplify CLI code base has been updated to reflect this change. Amplify CLI replaces Node.js 8.10 with Node.js 10 in the Lambda functions that it creates for you. If you use Amplify CLI version 4.10.0 and above to create new aws resources, this does not concern you.
+
+However, if you have used previous versions of the Amplify CLI to create aws resources in the following categories, 
+you will need to manually update your project to avoid Node.js runtime issues with AWS Lambda. 
+- auth
+- interactions
+
+Before you make the following manual changes, please make sure to back up your entire project. 
+
+After you make the following manual changes, run `amplify push` to update the AWS Lambda functions in the cloud. 
+
+### auth
+Auth allows you to add/configure Lambda Triggers for cognito, such as PostAuthentication and PostConfirmation, etc.. 
```

Comment:
1) Auth -> Auth category 
2)  such as PostAuthentication and PostConfirmation using `amplify add/update auth` command -> remove etc...